Scripts/Playthrough.cs

```diff
@@ -331,5 +350,29 @@ public void Play()
         }
 
         #endregion
+
+        #region Private functions
+
+        private bool IsReadyToPlay()
+        {
+            return _connectionState == ConnectionState.Connected;
+        }
```

Comment:
Duplicates `IsConnected` below, suggest removing `IsReadyToPlay` in favour of `IsConnected`